modules/rasBidAdapter.js

```diff
@@ -129,6 +129,36 @@ const getGdprParams = (bidderRequest) => {
   return queryString;
 };
 
+const parseAuctionConfigs = (serverResponse, bidRequest) => {
+  if (isEmpty(serverResponse) || isEmpty(bidRequest)) {
+    return null;
+  }
+  let auctionConfigs = [];
+  const bidConfigs = Object.fromEntries(bidRequest.bidIds.map(x => [x.bidId, x]));
+
+  serverResponse.ads.filter(bid => bidConfigs.hasOwnProperty(bid.id) && bidConfigs[bid.id].fledgeEnabled).forEach((bid) => {
+    auctionConfigs.push({
+      'bidId': bidConfigs[bid.id].bidId,
+      'config': {
+        'seller': 'https://csr.onet.pl',
+        'decisionLogicUrl': `https://csr.onet.pl/${bidConfigs[bid.id].network}/v1/protected-audience-api/decision-logic.js`,
+        'interestGroupBuyers': ['https://csr.onet.pl'],
+        'auctionSignals': {
+          'site': bidConfigs[bid.id].site,
+          'kvismobile': bidConfigs[bid.id].isMobile,
```

Comment:
we might be putting this in auction signals for you

Reply:
Do you mean putting in the parameters from adunit?

Reply:
nice, so we're filling in the prebid win price here and adding more stuff too https://github.com/prebid/Prebid.js/pull/10393

Reply:
would love your comments on #10419

Reply:
Ah, I see what you mean. This does look very useful. I'll try to look into #10419 soon, but I'll need to check what data we are using now and what could possibly be included.